src/tutorial/index.md

```diff
@@ -14,7 +14,7 @@ operation, see the manual portion of this documentation.
 
 How do you want to use TaskCluster?
 
- * [I'm not sure - what is it??](tutorial/what-is-tc)
```

Comment:
@djmitche oh, I thought this was a "todo title" kinda of note... :) lol... I never read the question, just the bullet points, and they seemed weird, no wonder... :)